contracts/FeeManager/FeeManager.sol

```diff
@@ -15,25 +16,33 @@ contract FeeManager is IFeeManager, Ownable {
     RIF private _rif;
 
     mapping(address => uint256) public balances;
```

Comment:
Have we decided on the implications or lack of thereof for leaving the balances public?

Reply:
Yeah I need to rebase